department-of-veterans-affairs/caseflow
# Adds new full page schedule veteran component

Resolves #14892 

### Description
Adds the new full-page schedule veteran component to setup the rest of the work to schedule veterans directly to virtual hearings

### Acceptance Criteria

### Testing Plan
1. Run storybook locally: `make storybook`
1. Navigate to the schedule veteran component `Hearings/Components/Schedule Veteran`
1. Ensure that all scenarios match the Figma diagram here: https://www.figma.com/proto/V87TZArfdurCGJiEjQ73ES/Virtual-Hearings?node-id=5896%3A24704&viewport=-269%2C32%2C0.06443598121404648&scaling=min-zoom

### User Facing Changes

 BEFORE|AFTER
 ---|---
![BeforeScheduleVeteran](https://user-images.githubusercontent.com/61989526/89915878-c4734280-dbab-11ea-8f65-7095fedffc96.png)|![AfterScheduleVeteran](https://user-images.githubusercontent.com/61989526/89915896-c937f680-dbab-11ea-90f3-659171a26289.png)


### Storybook Story
*For Frontend (Presentationa) Components*
